test/docs/get-docs-yaml.api-test.ts

```diff
@@ -0,0 +1,30 @@
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
+
+describe('GET /docs-yaml', () => {
+  let api: Api;
+
+  beforeAll(async () => {
+    api = await Api.create();
+  });
+
+  afterAll(async () => {
+    await api.destroy();
+  });
+
+  it('returns a 200 OK response', async () => {
+    const { status } = await api.getWithBasicAuth('/docs-yaml', {
+      username: ENVIRONMENT_VARIABLES.SWAGGER_USER,
```

Comment:
FYI, we are only using this locally to check the documentation and the swagger.json file that's generated.
In Azure we will end up using something like this -> https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-developer-portal

Reply:
Thanks @IgnatG, yep I knew we were going to use the Azure developer portal - am I right in assuming that we'll be importing the `yaml` file that our app generates into Azure APIM? If so then I still think it's worth testing our `yaml` file in this way

Reply:
that's right, either .yml file or .json file. 
and yes, happy to test it here